cmd/gearcmd/main.go

```diff
@@ -28,6 +29,8 @@ func main() {
 	retryCount := flag.Int("retry", 0, "Number of times to retry the job if it fails")
 	warningLength := flag.Int("warningLength", 5, "Number of warning lines to store and send back to the gearmn job")
 	passSigterm := flag.Bool("pass-sigterm", false, "Whether or not to pass SIGTERM through to the worker process")
+	errorBackoffCount := flag.Int("error-backoff-count", 5, "How many errors in a row before we wait before erroring jobs")
+	errorBackoffRate := flag.Duration("error-backoff-rate", 0, "How much time to sleep if last 'error-backoff-count' jobs have failed, e.g. 500ms, 1s")
```

Comment:
I would consider possible making this a minor version bump and putting something non-zero here. That'll make the rollout much less painful since we won't need to change the docker `CMD`.

Reply:
👍 I'll make it 5 seconds